weave/trace_server/trace_server_interface.py

```diff
@@ -915,6 +933,7 @@ def calls_query_stream(self, req: CallsQueryReq) -> Iterator[CallSchema]: ...
     def calls_delete(self, req: CallsDeleteReq) -> CallsDeleteRes: ...
     def calls_query_stats(self, req: CallsQueryStatsReq) -> CallsQueryStatsRes: ...
     def call_update(self, req: CallUpdateReq) -> CallUpdateRes: ...
+    def call_start_batch(self, req: CallCreateBatchReq) -> CallCreateBatchRes: ...
```

Comment:
The various "no-op" `call_start_batches` gives the impression that this is not correct. Why are they no-ops? Seems like an unnecessary coupling of fastapi logic and lower level interface logic. I think either:
a) there should not be a `call_start_batch` here; or
b) they implementations should be real.

Also, probably should call this `call_batch` since it is not limited to start events

Reply:
Alternatively: I think you could benefit from defining a `TraceServerAPI(tsi.TraceServerInterface)` that has additional functions (for example `call_start_batch`) if you did not want to couple the tsi interface with the service api - allowing the service api to be a true superset

Reply:
We can implement them for all the servers, I just decided it was a bit too much for one PR.

The name reflects what it's currently called in `core`, though we can change both.

Why would you want a `TraceServerAPI`?  It seems like going straight to `TraceServerInterface` is preferred